keras/layers/convolutional.py

```diff
@@ -703,6 +711,9 @@ class Conv2DTranspose(Conv2D):
         `(batch, new_rows, new_cols, filters)`
         if `data_format` is `"channels_last"`.
         `rows` and `cols` values might have changed due to padding.
+        If `output_padding` is specified:<br/>
```

Comment:
Please do not include html markup in the docstrings. Two line breaks should be sufficient (or use markdown list formatting).

Reply:
I think two line breaks have too much empty space. A two line break at  "If `output_padding` is specified:" and the equations inside a code block looks better to me. What's your preference?

Reply:
Docstrings should not contain HTML markup. Use plain markdown.